### PR TITLE
GA Promotion - Watermark for OnlineMeeting

### PR DESCRIPTION
**Pull request by yuyaolian-msft**

### CODE REVIEW

**Comment by yuyaolian-msft** [comment]
@dodaromike  Hi we typically truncate responses for those APIs as the body will get too long otherwise. (the documents have note saying the response is truncated). This has been discussed with another reviewer during the BETA documentation of this API.  Please see https://github.com/microsoftgraph/microsoft-graph-docs/pull/19629

image

**Comment by dodaromike** [comment]
@yuyaolian-msft Ok, I see that those examples have been shortened.  You are right.

**Review by dodaromike — APPROVED** [review]

**Review by dodaromike — APPROVED** [review]

**Review by dodaromike — APPROVED** [review]

File: api-reference/beta/resources/watermarkprotectionvalues.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Namespace: microsoft.graph
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-Represents the reasons for a participant's restricted media experience.
+Represents the reasons for a participant's restricted media experience. Please note this is not a security feature because it relies on the client application to implement the watermarking.
```
Comment (Lauragra) [inline api-reference/beta/resources/watermarkprotectionvalues.md:16]: Is this an accurate description? From the properties, it seems like the description here might be something like "Indicates whether a watermark should be applied to any content type.'

Why do we need to say that this is not a security feature? Can we simply say here: "The watermark is implemented by the client application."

File: api-reference/beta/resources/onlinemeeting.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ This resource supports subscribing to [change notifications](/graph/webhooks). S
 | startDateTime | DateTime | The meeting start time in UTC. |
 | subject | String | The subject of the online meeting. |
 | videoTeleconferenceId | String | The video teleconferencing ID. Read-only. |
-| watermarkProtection | [watermarkProtectionValues](watermarkprotectionvalues.md)     | Specifies whether a watermark applies for different entities. |
+| watermarkProtection | [watermarkProtectionValues](watermarkprotectionvalues.md)     | Specifies whether a watermark applies for different entities using the client appilcation. |
```
Comment (Lauragra) [inline api-reference/beta/resources/onlinemeeting.md:62]: Would a more accurate description here be: "Specifies whether a watermark should be applied to a content type by the client application."

File: api-reference/v1.0/resources/onlinemeeting.md [hunk 3]
```diff
@@ -51,6 +51,7 @@ Contains information about a meeting, including the URL used to join a meeting,
 | startDateTime         | DateTime                                      | The meeting start time in UTC.                                                                                             |
 | subject               | String                                        | The subject of the online meeting.                                                                                         |
 | videoTeleconferenceId | String                                        | The video teleconferencing ID. Read-only.                                                                                  |
+| watermarkProtection | [watermarkProtectionValues](watermarkprotectionvalues.md)     | Specifies whether a watermark applies for different entities using client appilcation. |                                                                |
```
Comment (Lauragra) [inline api-reference/v1.0/resources/onlinemeeting.md:54]: Would a more accurate description here be: "Specifies whether a watermark should be applied to a content type by the client application."

File: api-reference/v1.0/resources/watermarkprotectionvalues.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+doc_type: resourcePageType
+---
+
+# watermarkProtectionValues resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents the reasons for a participant's restricted media experience. Please note this is not a security feature because it relies on the client application to implement the watermarking.
```
Comment (Lauragra) [inline api-reference/v1.0/resources/watermarkprotectionvalues.md:16]: Is this an accurate description? From the properties, it seems like the description here might be something like "Indicates whether a watermark should be applied to any content type.'

Why do we need to say that this is not a security feature? Can we simply say here: "The watermark is implemented by the client application."

File: api-reference/beta/resources/watermarkprotectionvalues.md (hunk 1 above)
Comment (yuyaolian-msft) [reply api-reference/beta/resources/watermarkprotectionvalues.md:16]: Hi @Lauragra 
1. I will update the description
2. about the security feature note, we have been asked by the graph reviewers specifically to add this note 
image
https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding/pullrequest/13205?_a=files

File: api-reference/v1.0/resources/onlinemeeting.md (hunk 3 above)
Comment (yuyaolian-msft) [reply api-reference/v1.0/resources/onlinemeeting.md:54]: updated per your suggestion.

File: api-reference/v1.0/resources/watermarkprotectionvalues.md (hunk 4 above)
Comment (yuyaolian-msft) [reply api-reference/v1.0/resources/watermarkprotectionvalues.md:16]: Please see comment in the beta/resources/watermarkprotectionvalues.md

**Pull request merged**